app/models/CommonModels.scala

```diff
@@ -32,11 +32,14 @@ object SelectTitle extends Enumerable.Implicits {
   case object Mrs extends WithName("0002") with SelectTitle
   case object Miss extends WithName("0003") with SelectTitle
   case object Ms extends WithName("0004") with SelectTitle
+  case object UnsupportedTitle extends WithName("----") with SelectTitle
```

Comment:
would be good to have more meaningful string instead of "----" e.g. unsupported

Reply:
on it